### ISSUE TITLE
Update still failing (continued)

### ISSUE THREAD

**Issue by damagu** [SickChill/SickChill#6534]
Hi miigotu, 

Continued from [We can't proceed with the update. Unable to check remote DB version. #6510](https://github.com/SickChill/SickChill/issues/6510) which has been closed.  

I edited the Dockerfile, docker-compose.yaml and sickbeard/processTV.py files as per the link you provided in #6510, restarted sickchill and tried to update again. This time I got:

_Update Failed
Update wasn't successful, not restarting. Check your log for more information._

Branch: master
Commit: d21a4fc
Version: v2020.05.30-1
Database Version: 44.4
OS: Linux-3.14.29-157-osmc-aarch64-with-debian-9.12

Logs:
[sickchill.log](https://github.com/SickChill/SickChill/files/4778199/sickchill.log)

```
2020-06-15 12:50:38 DEBUG    CHECKVERSION :: Proceeding with auto update
2020-06-15 12:50:38 INFO     CHECKVERSION :: Config backup in progress...
2020-06-15 12:51:48 INFO     CHECKVERSION :: Config backup successful, updating...
2020-06-15 12:51:48 DEBUG    CHECKVERSION :: Executing git symbolic-ref -q HEAD with your shell in /opt/sickchill
2020-06-15 12:51:48 DEBUG    CHECKVERSION :: git symbolic-ref -q HEAD : returned successful
2020-06-15 12:51:48 DEBUG    CHECKVERSION :: Executing git rev-parse HEAD with your shell in /opt/sickchill
2020-06-15 12:51:48 DEBUG    CHECKVERSION :: git rev-parse HEAD : returned successful
2020-06-15 12:51:48 DEBUG    CHECKVERSION :: Executing git config remote.origin.url https://github.com/SickChill/SickChill.git with your shell in /opt/sickchill
2020-06-15 12:51:48 DEBUG    CHECKVERSION :: git config remote.origin.url https://github.com/SickChill/SickChill.git : returned successful
2020-06-15 12:51:48 DEBUG    CHECKVERSION :: Executing git fetch origin --prune with your shell in /opt/sickchill
2020-06-15 12:51:51 DEBUG    CHECKVERSION :: git fetch origin --prune : returned successful
2020-06-15 12:51:51 DEBUG    CHECKVERSION :: Executing git branch --set-upstream-to origin/master with your shell in /opt/sickchill
2020-06-15 12:51:53 DEBUG    CHECKVERSION :: git branch --set-upstream-to origin/master : returned successful
2020-06-15 12:51:53 DEBUG    CHECKVERSION :: Executing git rev-parse --verify --quiet "@{upstream}" with your shell in /opt/sickchill
2020-06-15 12:51:53 DEBUG    CHECKVERSION :: git rev-parse --verify --quiet "@{upstream}" : returned successful
2020-06-15 12:51:53 DEBUG    CHECKVERSION :: Executing git rev-list --left-right "@{upstream}"...HEAD with your shell in /opt/sickchill
2020-06-15 12:51:53 DEBUG    CHECKVERSION :: git rev-list --left-right "@{upstream}"...HEAD : returned successful
2020-06-15 12:51:53 DEBUG    CHECKVERSION :: cur_commit = d21a4fceaa4933e4d8ebc71cd0238d181988aebe, newest_commit = 806299598d8db36af8249147e2536af4f0303e20, num_commits_behind = 33, num_commits_ahead = 0
2020-06-15 12:51:53 DEBUG    CHECKVERSION :: Executing git config remote.origin.url https://github.com/SickChill/SickChill.git with your shell in /opt/sickchill
2020-06-15 12:51:53 DEBUG    CHECKVERSION :: git config remote.origin.url https://github.com/SickChill/SickChill.git : returned successful
2020-06-15 12:51:53 DEBUG    CHECKVERSION :: Executing git reset --hard with your shell in /opt/sickchill
2020-06-15 12:51:53 DEBUG    CHECKVERSION :: git reset --hard : returned successful
2020-06-15 12:51:53 DEBUG    CHECKVERSION :: Executing git symbolic-ref -q HEAD with your shell in /opt/sickchill
2020-06-15 12:51:53 DEBUG    CHECKVERSION :: git symbolic-ref -q HEAD : returned successful
2020-06-15 12:51:53 DEBUG    CHECKVERSION :: Executing git pull -f origin master with your shell in /opt/sickchill
2020-06-15 12:51:56 INFO     CHECKVERSION :: Update failed!

```

**Comment by miigotu** [comment]
> I edited the Dockerfile, docker-compose.yaml and sickbeard/processTV.py files as per the link you provided in #6510, restarted sickchill and tried to update again. This time I got:

@damagu you weren't supposed to edit anything. Those were changes I made to the docker image on master.

You should be updating with watchtowerrr if updates do not work inside SC.

![image](https://user-images.githubusercontent.com/390379/84626687-8ad7d100-aeb3-11ea-82dc-c031bca304af.png)

**Comment by miigotu** [comment]
@damagu I might be confused as well. Do you have it installed via docker? If not, stop SC and use git to update it.

```bash
cd /path/to/sickchill
git fetch origin
git reset --hard origin/master
git pull
```

**Comment by miigotu** [comment]
Also, you can still comment in closed issues. I will close this one because it is not an issue with the app but something up on your install. We can continue to discuss in this same issue.

**Comment by damagu** [comment]
Hi miigotu, 

Sorry. I assumed that because you closed the issue that it meant that as far you were concerned the issue was resolved. Changes you make in master have no effect on my install until I'm able to update again, right? So when you pointed to the changes you made and said:

> The unable to contact github might be due to not having the correct timezone inside the container, which is solved by:
> 80abda0#diff-3254677a7917c6c01f55212f86c57fbfR10

... I also assumed that "which is solved by" meant I would need to solve the problem by making the indicated changes. 

As the saying goes: assumption makes an "ass" out of "u" and "mption"... or something like that :). So... my bad!

Anyway...

> Do you have it installed via docker?

No... at least I don't think so.  I installed sickchill on this particular device (a vero4k running OSMC) using [this guide](https://discourse.osmc.tv/t/how-to-install-couchpotato-and-sickchill-on-raspberry-pi/10788) which makes no mention of docker.

>  If not, stop SC and use git to update it.

Ok I did that and ran the commands as instructed then restarted sickchill and I'm no longer seeing the messages about there being available updates / being x commits behind... so I guess that means it worked. 

Thank you. Appreciate your effort.
